services/workflows-service/src/case-management/types/transition-schema.ts

```diff
@@ -0,0 +1,5 @@
+export interface ITransitionSchema {
```

Comment:
transition schema in case managment types? should be in the workflow lib or at least workflow type in WF service